src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/DeclarationTests.cs

```diff
@@ -146,14 +147,16 @@ .maxstack  4
   IL_0044:  ldfld      ""int System.ValueTuple<int, string>.Item1""
   IL_0049:  stloc.1
   IL_004a:  ldfld      ""string System.ValueTuple<int, string>.Item2""
-  IL_004f:  ldstr      ""z1""
-  IL_0054:  call       ""int Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<int>(string)""
-  IL_0059:  ldloc.1
-  IL_005a:  stind.i4
-  IL_005b:  ldstr      ""z2""
-  IL_0060:  call       ""string Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<string>(string)""
-  IL_0065:  stind.ref
-  IL_0066:  ret
+  IL_004f:  stloc.2
+  IL_0050:  ldstr      ""z1""
+  IL_0055:  call       ""int Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<int>(string)""
+  IL_005a:  ldloc.1
+  IL_005b:  stind.i4
+  IL_005c:  ldstr      ""z2""
+  IL_0061:  call       ""string Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<string>(string)""
+  IL_0066:  ldloc.2
```

Comment:
Looking into this EE IL question more, it definitely seems the previous IL is incorrect. The `stind` instruction needs an address and then a value (on top), but here there was no value on the stack.
I'll investigate.